source/server/admin/admin.cc

```diff
@@ -619,24 +652,59 @@ ProtobufTypes::MessagePtr AdminImpl::dumpEndpointConfigs() const {
   return endpoint_config_dump;
 }
 
+ProtobufTypes::MessagePtr AdminImpl::dumpUnreadyTargetsConfigs(bool dump_active_listeners) const {
```

Comment:
We only need dump all targets. The targets could comes from
1. `active listeners`, if workers are not started yet.
2.  `warm listeners`, if workers are started.